tools/aiecc/aiecc/main.py

```diff
@@ -118,7 +118,7 @@ async def prepare_for_chesshack(self, task):
       if(opts.compile and opts.xchesscc):
         thispath = os.path.dirname(os.path.realpath(__file__))
         # Should be architecture-specific ?
-        runtime_lib_path = os.path.join(thispath, '..','..','runtime_lib')
+        runtime_lib_path = os.path.join(thispath, '..','..')
         chess_intrinsic_wrapper_cpp = os.path.join(runtime_lib_path, 'chess_intrinsic_wrapper.cpp')
```

Comment:
This is wrong.  There must be something borked in cmake?  Are you using a build directory or an install directory?

Reply:
The install directory (after a fresh build using 2022.2). chess_intrinsic_wrapper.cpp, test_library.cpp, test_library.h are all directly in the install directory.

Reply:
https://github.com/Xilinx/mlir-aie/pull/371